src/server/game/Server/Packets/LFGPackets.cpp

```diff
@@ -36,3 +36,354 @@ ByteBuffer& operator<<(ByteBuffer& data, WorldPackets::LFG::RideTicket const& ti
 
     return data;
 }
+
+ByteBuffer& operator>>(ByteBuffer& data, WorldPackets::LFG::LfgBootInfo& bootInfo)
+{
+    data >> bootInfo.VoteInProgress;
```

Comment:
You may be right. Not at the computer ATM, will check when I can. I don't remember. It's like 5 am lmao.